.github/workflows/build-arm64-wheels.yml

```
@@ -23,99 +23,117 @@ jobs:
       image: ghcr.io/chia-network/build-images/centos-pypa-rust-aarch64:latest
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v3
-      with:
-        fetch-depth: 1
-
-    - name: Set up
-      run: |
-        echo "${PATH}"
-        yum -y install openssl-devel
-        source /root/.cargo/env
-        rustup default stable
-        rustup target add aarch64-unknown-linux-musl
-        rm -rf venv
-        export PATH="${PATH}:/opt/python/cp39-cp39/bin/:/opt/python/cp38-cp38/bin/:/opt/python/cp37-cp37m/bin/"
-    - name: Build Python wheels
-      run: |
-        /opt/python/cp38-cp38/bin/python -m venv venv
-        if [ ! -f "activate" ]; then ln -s venv/bin/activate; fi
-        . ./activate
-        pip install maturin==1.1.0
-        CC=gcc maturin build --release --strip --manylinux 2014
-    - name: Upload artifacts
-      uses: actions/upload-artifact@v3
-      with:
-        name: wheels
-        path: target/wheels/
-
-    - name: Clean up AMR64
-      run: |
-        rm -rf venv
-        rm -rf dist
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Set up
+        run: |
+          echo "${PATH}"
+          source /root/.cargo/env
+          rustup default stable
+          rustup target add aarch64-unknown-linux-musl
+          rm -rf venv
+
+      - name: Build Python wheels
+        run: |
+          /opt/python/cp38-cp38/bin/python -m venv venv
+          if [ ! -f "activate" ]; then ln -s venv/bin/activate; fi
+          . ./activate
+          pip install --upgrade pip
+          pip install maturin==1.1.0
+          CC=gcc maturin build --release --strip --manylinux 2014
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: target/wheels/
+
+      - name: Clean up ARM64
+        run: |
+          rm -rf venv
+          rm -rf dist
+
+  cargo_test:
+    name: Run cargo test on ARM64 Linux
+    runs-on: [ARM64, Linux]
+    container:
+      image: ghcr.io/chia-network/build-images/centos-pypa-rust-aarch64:latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
v4 is current version of this. would be nice to update while we're in here doing a bunch of changes anyways.